### PR TITLE
AB React

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What does this change?
Adds the `ab-react` package to provide react specific functions for using ab tests

### CODE REVIEW

File: packages/ab-react/src/types.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
+export type ConfigType = {
```
Comment (gtrufitt) [inline packages/ab-react/src/types.ts:1]: I guess we'd want to share the types or import them from core

**Comment by gtrufitt** [comment]
Ah ok, so you're still using lerna here rather than published

**Comment by oliverlloyd** [comment]
> Ah ok, so you're still using lerna here rather than published

Yes, although not my intention it has advantages so I'm willing to go with the tide

File: packages/ab-react/src/types.ts (hunk 1 above)
Comment (oliverlloyd) [reply packages/ab-react/src/types.ts]: Actually, non of these were being used and were copied here in error. Removed

File: packages/ab-react/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# ab-rendering
```
Comment (gtrufitt) [inline packages/ab-react/README.md:1]: Note, I'll update these docs in a later PR

**Pull request merged**